### PR TITLE
Added validation message for checking Service port and protocol consistency.

### PR DESCRIPTION
**Pull request by hhovsepy**
RFE https://github.com/kiali/kiali/issues/4782

### CODE REVIEW

File: content/en/docs/Features/validations.md [hunk 1]
```diff
@@ -569,6 +569,30 @@ Rename the service port appProtocol field to follow the form and the traffic flo
...
+
+#### Example
+
+```yaml
+{{% readfile file="/static/files/validation_examples/603.yaml" %}}
+```
+
+#### See Also
+
+- [Validator source code](https://github.com/kiali/kiali/blob/v1.42.0/business/checkers/services/port_mapping_checker.go)
```
Comment (jshaughn) [inline content/en/docs/Features/validations.md]: Perhaps for new validator the version should be v1.51.0 as opposed to the older v1.42.0 ?

**Review by jshaughn — APPROVED** [review]
Looks good, one non-blocking comment.  Also, I think you probably want to re-target to 'staging', and perhaps backport to 'current', if necessary.

File: content/en/docs/Features/validations.md (hunk 1 above)
Comment (hhovsepy) [reply content/en/docs/Features/validations.md]: or, can it point to upcoming v1.52.0 branch?

Comment (jshaughn) [reply content/en/docs/Features/validations.md]: It could, but only if this is for 'staging', that would be invalid for 'current'.

Comment (jshaughn) [reply content/en/docs/Features/validations.md]: Still looks good, merge if you like.

**Comment by hhovsepy** [comment]
Closing this PR as the main PR is closed for further investigation.

**Pull request closed without merging**